satellite/metainfo/metainfo.go

```diff
@@ -77,7 +77,7 @@ func (endpoint *Endpoint) validateAuth(ctx context.Context) (*console.APIKeyInfo
 		return nil, status.Errorf(codes.Unauthenticated, "Invalid API credential")
 	}
 
-	key, err := console.APIKeyFromBase64(string(APIKey))
+	key, err := console.APIKeyFromBase32(string(APIKey))
```

Comment:
Let's make `auth.GetAPIKey` already do this conversion, makes the code bit shorter.

Reply:
Like to make `GetAPIKey` return `console.APIKey`?

Reply:
Yeah, it seems weird that you need to decode the API key that is fetched from context.

PS: can be a separate PR.